shell/.eslintrc.yml

```diff
@@ -0,0 +1,11 @@
+---
+parser: "babel-eslint"
```

Comment:
Any reason in particular for choosing `babel-eslint` over the default `espree` parser vs going straight to `@typescript-eslint/parser`?

Reply:
The default didn't work because there are a couple places where we're doing `if(Meteor.isServer) { import ... }` and it refuses to parse that (it's apparently not technically legal according to the spec).

I didn't try the typescript one; maybe I'll see how that goes.

Reply:
Ahhh, seems reasonable.  I'll merge this; feel free to switch to the typescript one in a followup patchset.